integration-tests/src/test/resources/integration-test/distro-properties/openmrs-distro.properties

```diff
@@ -0,0 +1,8 @@
+#Fri Dec 30 10:54:43 CET 2016
+version=2.5
+name=Reference Application
+distro.referenceapplication.artifactId=referenceapplication-package
```

Comment:
We want SDK to translate referenceapplication to referenceapplication-package in code automatically so that this line is not needed.